data/help.txt

```diff
@@ -128,3 +128,10 @@ help "trading"
 
 help "friendly disabled"
 	`This ship is disabled and needs your help to patch them up! Pressing <Board selected ship> will fly you to their ship and get them up and running again. Just be sure to do it when no one is trying to kill you, or they could get caught in the crossfire!`
+
+help "tributing"
+	`You are about to tribute a planet. When tributing a planet you will have to battle against a defense fleet and if you win you will have unrestricted access to this planet forever and will receive a small amount of money every day. Tributing is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want the canon experience of this game you should not tribute a planet.`
```

Comment:
```suggestion
	`You are about to demand tribute from a planet. When demanding tribute from a planet, you will have to defeat it's defense fleet, and if you win you will have unrestricted access to this planet forever and receive a small amount of credits every day. Demanding tribute is an immersion breaking action that can have negative consequences on campaigns or mission lines. If you want an unaltered experience of this game, you should not demand tribute from a planet.`
```
`Tributing` sounds like you're the one giving tribute. I'm pretty sure `tribute` is not a verb, `to tribute` doesn't sound right.

Reply:
it's defense fleet -> its